lib/apartment/adapters/abstract_adapter.rb

```diff
@@ -218,6 +224,17 @@ def load_or_raise(file)
       # Backward compatibility
       alias_method :load_or_abort, :load_or_raise
 
+      #   Load a SQL file and execute it or abort if it doesn't exists
+      #
+      def execute_or_abort(file)
+        if File.exists?(file)
```

Comment:
```ruby
abort "#{file} doesn't exist yet" unless File.exists? file
```

Might read a bit nicer

Reply:
+1, gonna opt to leave this in, since it is from the original external PR